bin/run_lint

```diff
@@ -260,22 +276,53 @@ top_dir=$(cd $(dirname $(dirname $0)) && pwd)
 rv=0
 
 # Run pycodestyle or pylint or both for each module.
-# Modules should be ranked in decreasing likelihood of having lint errors.
 
 # Language:                                Python  | C
 # Checks:                           pycodestyle    | pycodestyle
 #                                           pylint |    cppcheck
 export PYTHONPATH=$PYTHONPATH:$top_dir/examples/common/python
```

Comment:
$top_dir/examples/common/python is not valid any more. Should be updated

Reply:
Not needed as all installations using pip3 goes to default dist-packages